cryptocurrency/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM ghcr.io/deephaven/grpc-api
+COPY cryptoApp.app /
+COPY crypto.py /
+run pip3 install pycoingecko
```

Comment:
I'm not sure if Dockerfile commands *need* to be capitalized, but by convention they *should*. `run` -> `RUN`.